### ISSUE TITLE
Noto Naskh Arabic - More Weights?

### ISSUE THREAD

**Issue by MAZ06** [notofonts/arabic#233]
It seems that Noto Naskh Arabic does not have as many weights as the Sans and Kufi styles. Am I just not looking in the right place or does Naskh only have Regular, Medium, SemiBold, and Bold weights? Are there any plans on bringing more weights to Noto Naskh Arabic?

**Comment by simoncozens** [comment]
You're correct, Noto Naskh Arabic only supplies weights 400-700, whereas the other fonts include more weights. There aren't any plans to add more weights at the moment, though, as there's nothing we need them for.
